lib/books/book.rb

```diff
@@ -0,0 +1,9 @@
+require_relative '../item'
+
+class Book < Item
+  def initialize(publisher, cover_state)
```

Comment:
- [ ] Please add all the parameters needed to create a new instance of the book. Book is a child of the Item class and must be created with all parameters required for the Item class. Parameters that have default values don't need to be specified. To create a new instance of a Book is need to inform data such as Label, Source, Author, and Genre.